build_tools/build_release.py

```diff
@@ -55,13 +63,13 @@ def write_version_info(args):
     with open(VERSION_INFO_FILE, "rt") as f:
         info_dict = json.load(f)
 
-    # Compute core-version.
-    core_version = eval_version(args.core_version)
-    if args.core_pre_version:
-        core_version += eval_version(args.core_pre_version)
-    if args.core_post_version:
-        core_version += f".{eval_version(args.core_post_version)}"
-    info_dict["core-version"] = core_version
+    # Compute package-version.
+    package_version = eval_version(args.package_version)
+    if args.package_pre_version:
+        package_version += eval_version(args.package_pre_version)
+    if args.package_post_version:
+        package_version += f".{eval_version(args.package_post_version)}"
+    info_dict["package-version"] = package_version
```

Comment:
Yet another way to compute / parse the version numbers. I assume we want to harmonize this over the projects when switching iree-turbine to the new versioning scheme? Nothing for this PR, just considerations.